tests/Console/ConfigurationResolverTest.php

```diff
@@ -379,6 +379,47 @@ public function testResolvePathRelativeB()
         $this->assertSame(array(__DIR__), $resolver->getPath());
     }
 
+    /**
+     * @param int      $expectedCount
+     * @param string[] $paths
+     *
+     * @dataProvider provideDoublePaths
+     */
+    public function testResolveDoublePaths($expectedCount, array $paths)
+    {
+        $resolver = new ConfigurationResolver(
+            $this->config,
+            array(
+                'path' => $paths,
+                'config' => __DIR__.'/../Fixtures/ConfigurationResolverConfigFile/case_4/my.php_cs',
+            ),
+            dirname(__DIR__)
+        );
+
+        $this->assertCount($expectedCount, $resolver->getPath());
```

Comment:
Wouldn't it be better to check the exact result (i.e. `assertSame()`)? I think having only the count assertion makes the test weak.